R/diagnostics.R

```diff
@@ -19,6 +19,10 @@
 #'   tolerance set by {tol}, FALSE otherwise.
 #' @export
 isTimestepRegular <- function(dates, hist=TRUE, tol=.Machine$double.eps^0.5, handler=stop) {
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes=TimeInterval=geom_histogram=xlab=NULL
```

Comment:
great - glad the @importFrom thing helped.

I don't understand your suggestion about ??.TimeInterval ?. Could you paste the full code change you're suggesting?